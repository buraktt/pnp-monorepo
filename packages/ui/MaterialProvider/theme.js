import { createMuiTheme } from '@material-ui/core/styles'

const theme = createMuiTheme({
  typography: {
    useNextVariants: true,
    fontFamily: 'Montserrat',
    fontSize: 14,
    htmlFontSize: 14
  }
})

export default theme
