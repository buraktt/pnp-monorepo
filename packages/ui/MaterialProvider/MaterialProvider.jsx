import React from 'react'
import { MuiThemeProvider } from '@material-ui/core/styles'
import { MuiPickersUtilsProvider } from '@material-ui/pickers'
import DateFnsUtils from '@date-io/date-fns'
import { CssBaseline } from '@material-ui/core'
import theme from './theme'

export default function MaterialProvider(props) {
  const { children } = props
  return (
    <MuiPickersUtilsProvider utils={DateFnsUtils}>
      <MuiThemeProvider theme={theme}>
        <CssBaseline />
        {children}
      </MuiThemeProvider>
    </MuiPickersUtilsProvider>
  )
}
