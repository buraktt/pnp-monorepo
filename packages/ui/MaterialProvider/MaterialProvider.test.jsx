/**
 * @group snapshot
 */

import React from 'react'
import MaterialProvider from './MaterialProvider'
import { render } from '@testing-library/react'

describe('<MaterialProvider />', () => {
  it('should render without errors', () => {
    const { asFragment } = render(<MaterialProvider />)
    expect(asFragment()).toMatchSnapshot()
  })
})
