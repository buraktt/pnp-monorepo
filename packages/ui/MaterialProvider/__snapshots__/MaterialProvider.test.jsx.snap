// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MaterialProvider /> should render without errors 1`] = `<DocumentFragment />`;
